local/gugcat/renderer.php

```diff
@@ -238,7 +242,7 @@ private function display_table($rows, $columns) {
         return $html;
     }
 
-    private function context_actions($studentno, $ishidden=null, $is_aggregrade = false, $link = null) {
+    private function context_actions($studentno, $ishidden=null, $is_aggregrade = false, $is_ammendgrade = false, $link = null) {
```

Comment:
Was  $is_ammendgrade used? If not, please remove, and update affected codes.

Reply:
I used that to get the link for ammend grade

Reply:
I already remove that, and I'll push it